.github/workflows/ci.yaml

```diff
@@ -2300,4 +2300,7 @@ name: Cactus_CI
   push:
     branches:
       - main
-      - dev
\ No newline at end of file
+      - dev
+  schedule:
+    # Run at 8:00 AM UTC on weekends (Saturday and Sunday)
```

Comment:
@aldousalvarez Looking better, thank you for the updates! If you want to run it twice a week I'd recommend Monday and Thursday. Statistically speaking it should have a higher probability of catching things as early as possible compared to it running twice in relatively quick succession over the weekend. @outSH Thoughts?

Reply:
I was thinking about adding a separate workflow that would be run beside the CI, but this is OK too :)

As for the schedule, I agree to have wider gap between the runs, Monday and Thursday sounds great

Reply:
@petermetz, @outSH updated it to Monday and Thursday. Re-requested for review. Thank you